client/actions/lectureActions.js

```diff
@@ -7,9 +7,7 @@ export function getLectures(page) {
             .then((response) => {
                 dispatch(getLecturesAction(response.data))
             })
-            .catch((err) => {
-                console.log(err);
-            })
+            .catch({/** */})
```

Comment:
please don't swallow exceptions.
use this instead
catch((err) => { console.error(err);}